flink-connector-aws/flink-connector-dynamodb/src/main/java/org/apache/flink/connector/dynamodb/source/util/ShardUtils.java

```diff
@@ -26,7 +26,10 @@
 /** Utility methods for properties of DynamoDB streams shards. */
 @Internal
 public class ShardUtils {
-    private static final Duration DDB_STREAMS_MAX_RETENTION_PERIOD = Duration.ofHours(25);
+    private static final Duration DDB_STREAMS_MAX_RETENTION_PERIOD = Duration.ofHours(48);
+
+    private static final Duration DDB_STREAMS_MAX_RETENTION_PERIOD_FOR_RESOLVING_INCONSISTENCIES =
```

Comment:
added comment for the same

Reply:
Thanks for adding the comment @gguptp , but can we add some explanation for why we chose these values? This is to serve as reasoning for future implementers why we chose these values.


e.g. we set this to be 25 hours because records in DDB stream are expired after 24 hours, so we don't need to resolve inconsistencies in those shards, as they will be deleted.

And for the `48` hours, maybe we can say that we allow DDB to allow DDB shards to remain after maximum retention period.